src/InputType.hack

```diff
@@ -0,0 +1,15 @@
+namespace Slack\GraphQL;
+
+/**
+* GraphQL input type: https://graphql.org/learn/schema/#input-types
+*
+* Annotate shapes with this attribute to support accepting them as inputs within
+* mutations.
+*/
+class InputType implements \HH\TypeAliasAttribute {
```

Comment:
naming nitpick: "input type" can mean any of scalars, enums, input objects

Maybe "InputObjectType" or just "InputObject"?

The usage of "object" here can be a bit confusing for people who don't work in JavaScript; in JS "object" is basically a map/dict.

Reply:
I'll go with InputObjectType for now to stay consistent